.github/workflows/execution-plan-snippet.yml

```diff
@@ -88,12 +94,15 @@ jobs:
       options: --user root
       credentials:
         username: "_json_key"
-        password: ${{ secrets.FDC_CORE_CI_IMAGE_READER }}
+        password: ${{ secrets.GCP_ARTIFACT_REGISTRY_PUSH_JSON_KEY }}
```

Comment:
This is needed to be able to pull the `kuberpult-builder` image?

Reply:
removed it. public image so not required now